js/views/pageNavVw.js

```diff
@@ -66,6 +67,12 @@ module.exports = baseVw.extend({
       this.model.fetch();
     });
 
+    ipcRenderer.on('goto', function(event, arg) {
+      self.addressBarProcess(arg);
+    });
+
+    ipcRenderer.send('checkURI', '');
+
```

Comment:
This may be an issue.

main.js does the following in order

- creates pageNavVw, which fires this initialize function
- changes the location.hash to # or the value passed to loadProfile
- creates the router, which navigates to the route in local storage if the location hash is set to # 

It would be better if you added a if(external_uri) {} check in loadProfile to not set the location hash if an external url has been loaded.